src/app/core/components/breadcrumbs/breadcrumbs.component.scss

```diff
@@ -0,0 +1,11 @@
+.breadcrumb-container {
+    width: 100%;
+    height: 64px;
+    background-color: #fff;
+
+    li {
+        padding-top: 20px;
+        padding-left: 85px;
+        list-style-type: none;
```

Comment:
Could you please remove lists padding-left and make the breadcrumbs item text bold as it is shown in the mockup?